plugins/services/src/js/routes/services.js

```diff
@@ -171,44 +180,117 @@ const serviceRoutes = [
           },
           {
             type: Route,
+            path: "configuration",
+            title: "Configuration",
+            component: HighOrderServiceConfiguration,
+            buildBreadCrumb() {
```

Comment:
Oh yeah I don't think you should remove all of the `buildBreadCrumb` methods here, but shouldn't we avoid adding new ones, since they're not in use anyway?